### PR TITLE
Wrap functions in function_exists() checks to avoid fatals

### PR DESCRIPTION
**Pull request by borkweb**
Best viewed with `?w=1`

See: https://central.tri.be/issues/41134

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [general.php:26](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L26)
:warning: | Comment refers to a TODO task | [general.php:61](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L61)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$url' | [general.php:122](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L122)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$the_notices' | [general.php:308](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L308)
:warning: | Comment refers to a TODO task | [general.php:360](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L360)
:warning: | The method parameter $value is never used | [general.php:362](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L362)
:warning: | The method parameter $key is never used | [general.php:362](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L362)
:warning: | Comment refers to a TODO task | [general.php:375](https://github.com/moderntribe/tribe-common/blob/5d29fd6a51bca8115382703dadb8da0571b3eb22/src/functions/template-tags/general.php#L375)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -86,7 +86,6 @@ public function init_libraries() {
 
 		require_once $this->plugin_path . 'src/functions/template-tags/general.php';
 		require_once $this->plugin_path . 'src/functions/template-tags/date.php';
-		require_once $this->plugin_path . 'src/functions/template-tags/day.php';
```
Comment (zbtirrell) [inline src/Tribe/Main.php]: this include is no longer needed?

Comment (borkweb) [inline src/Tribe/Main.php]: Correct. That file is also removed in this commit. The contents of the file was:

```php
<?php
```

Comment (zbtirrell) [inline src/Tribe/Main.php]: wow.  that seems pretty important...

Comment (borkweb) [inline src/Tribe/Main.php]: We can all mourn its loss together.

:sob:

**Comment by zbtirrell** [comment]
:+1:

**Pull request merged**